.travis.yml

```diff
@@ -0,0 +1,24 @@
+sudo: required
+
+language: python
+
+python:
+  - 2.7
+
+services:
+  - docker
+
+env:
+  DOCKER_COMPOSE_VERSION: 1.10.0
+
+install: sudo pip install -r requirements.txt -t app/vendors
```

Comment:
Do you need this? The libraries are already in app/vendors directory of Github repository, right?